apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -1,41 +1,41 @@
 {{#question}}
 
+{{#radio}}
 {{#inline}}
 <span>
 {{/inline}}
-{{#answers}}
-    <div class="form-check {{#inline}}form-check-inline d-inline-flex{{/inline}}{{^inline}}d-flex{{/inline}} align-items-center py-1">
-        <input class="form-check-input mt-0" type="radio"
-               name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
-               {{#checked}}checked{{/checked}} id="{{name}}-{{key}}">
-
-        <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
-            {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
-            <div class="ml-1 mr-1">{{{html}}}</div>
-        </label>
-
-        {{#display_feedback}}
-        <div class="pl-multiple-choice-feedback">
-            {{{feedback}}}
+    {{#answers}}
+        <div class="form-check {{#inline}}form-check-inline d-inline-flex{{/inline}}{{^inline}}d-flex{{/inline}} align-items-center py-1">
+            <input class="form-check-input mt-0" type="radio"
+                name="{{name}}" value="{{key}}" {{^editable}}disabled{{/editable}}
+                {{#selected}}checked{{/selected}} id="{{name}}-{{key}}">
+
+            <label class="form-check-label d-flex align-items-center" for="{{name}}-{{key}}">
+                {{^hide_letter_keys}}<div class="pl-multiple-choice-key-label">({{key}})</div>{{/hide_letter_keys}}
+                <div class="ml-1 mr-1 pl-answer">{{{html}}}</div>
```

Comment:
Added this in an attempt to address https://github.com/PrairieLearn/PrairieLearn/issues/4391. Not sure what the best way is however, I'm also not sure whether we should add the `pl-answer` classes to what's displayed in the submission panel as well. Curious if @nwalters512 has a suggested path forward with this.

Reply:
Tbh, I'd rather do nothing at the moment until we can discuss an approach for consistent class names on things (if indeed we want to do that at all - I'm not yet sold on the idea that instructors should be overriding styles on our elements).

Reply:
That seems reasonable, I had similar concerns while I was looking at the original issue. I'll revert the changes and we can discuss later if needed.

Reply:
Removed classes here: https://github.com/PrairieLearn/PrairieLearn/pull/8903/commits/c6768304b435d2bd85b5a940abd27a2bf152a4be